integration/framework.go

```diff
@@ -457,3 +458,21 @@ func (f *framework) ApplyAWSConfigPatch(patch []PatchSpec, clusterName string) e
 
 	return nil
 }
+
+func (f *framework) GetVersionBundleVersion(bundle []versionbundle.Bundle, vType string) string {
```

Comment:
This doesn't have to be a method of framework, but this is a minor thing.

But, wouldn't it be nice to validate vType? Or if you don't want to return error make it a typed constant and handle it in TestMain?

Reply:
Sure, made function.

Ok, I've added validation and now the function returns an error too.